src/contracts/facilitators/gsm/Gsm.sol

```diff
@@ -527,6 +553,14 @@ contract Gsm is AccessControl, VersionedInitializable, EIP712, IGsm {
     return (finalAssetAmount, finalGrossAmount - finalFee, finalGrossAmount, finalFee);
   }
 
+  /**
+   * Returns the amount of GHO that has been sent out by the GSM
+   */
+  function _getGhoOutstanding() internal view virtual returns (uint256) {
```

Comment:
the name is not very clear to me. What about the following terms:
- getUsedGho: return the gho used by the contract
- getAvailableGho: return the gho limit that can be used by the contract
- useGho/restoreGho: transfers from/to the vault

Reply:
we can also use `use`/`restore` and just `mint/burn` as we are minting and burning in practice (putting liquidity into circulation)